### PR TITLE
Add ZF2016-02/CVE-2016-6233 for zendframework1

### PR DESCRIPTION
**Pull request by DavidPrevot**

### CODE REVIEW

File: zendframework/zendframework1/ZF2016-02.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+title:     Potential SQL injection in ORDER and GROUP statements of Zend_Db_Select
+link:      http://framework.zend.com/security/advisory/ZF2016-02
+cve:       CVE-2016-6233
+branches:
+    1.12.x:
+        time:     2016-07-06 17:01:58
+        versions: ['>=1.12.9', '<1.12.19']
```
Comment (xabbuh) [inline zendframework/zendframework1/ZF2016-02.yaml]: Are you sure about the lower bound? I didn't find any information on this in the blog post.

Comment (fabpot) [inline zendframework/zendframework1/ZF2016-02.yaml]: The lower bound should be removed as far I understand the blog post. Should be `<1.12.19` only. Anyway, the current versions constraint does not match anything.

Comment (fabpot) [inline zendframework/zendframework1/ZF2016-02.yaml]: I'm going to make the change after the merge.

Comment (DavidPrevot) [inline zendframework/zendframework1/ZF2016-02.yaml]: Le 18/07/2016 à 03:14, Christian Flothmann a écrit :
>> @@ -0,0 +1,8 @@
>> +title:     Potential SQL injection in ORDER and GROUP statements of Zend_Db_Select
>> +link:      http://framework.zend.com/security/advisory/ZF2016-02
>> +cve:       CVE-2016-6233
>> +branches:
>> +    1.12.x:
>> +        time:     2016-07-06 17:01:58
>> +        versions: ['>=1.12.9', '<1.12.19']
> 
> Are you sure about the lower bound?

Yes, the affected code was added at least starting with 1.12.10. Even
the proof of concept from the blog post fails with 1.12.9.

Regards

David

**Pull request merged**